Update commands and steps in Readme (#1719)
PrestaShop/autoupgrade: README.md

1. **UpdateInitialization**: Start of the whole process, clear potential leftover temporary files from a previous process. The next step will be chosen depending on the configuration.
2. **Download**: Download the appropriate archive for your PHP version.
3. **Unzip**: Unzip the downloaded archive.
4. **DownloadModules**: Find and download the most up-to-date release available for each installed modules. They will be applied after the store update.

> 5. **UninstallModules**: Uninstall all modules that are compatible with the updated version of PrestaShop. 

Reviewer: ```suggestion
5. **UninstallModules**: Uninstall all modules not compatible with the updated version of PrestaShop. 
```

Author: ```suggestion
5. **UninstallModules**: Uninstall all modules that are not compatible with the updated version of PrestaShop. 
```

6. **UpdateFiles**: Now the current content is saved, it can alter the store content. This step will run several time. The first call will initialize the files list, then the next ones will copy a part of this list.
7. **UpdateDatabase**: This step runs all the update SQL files available for the destination version. Then, it will run some additional steps, such as cache deletion, language update…

> 8. **UpdateModules**: This step updates the modules files and executes potential migrations from local sources, composer or from the Prestashop marketplace.

Reviewer: we can update this part because we are not downloaded it her anymore
